tensorflow_lite_support/ios/ios.bzl

```diff
@@ -28,3 +28,32 @@ def strip_c_api_include_path_prefix(name, hdr_labels, prefix = ""):
             > "$@"
             """.format(prefix, hdr_label),
         )
+
+# When the static framework is built with bazel, all the header files are moved
+# to the "Headers" directory with no header path prefixes. This auxiliary rule
+# is used for stripping the path prefix from the iOS API header files imported by
+# other iOS API header files.
+def strip_ios_api_include_path_prefix(name, hdr_labels, prefix = ""):
```

Comment:
Yes stripping will be needed. Right now only one file has been added for reviewing the structure. In a full implementation the header would have a lot of imports in a hierarchical structure like [TFLImageClassifier.h](https://github.com/tensorflow/tflite-support/blob/master/tensorflow_lite_support/ios/task/vision/sources/TFLImageClassifier.h#L17). We will have to strip the prefixes and build the final iOS framework like the C framework is built.
I added a separate method since objective c files are imported using `#import` as opposed to c files which are included using `#include`. I can incorporate this check in  `strip_c_api_include_path_prefix` and call it `strip_api_include_path_prefix` which will strip path prefixes irrespective of  `#import ` or `#include`

Reply:
I see. If it's needed, I'd recommend consolidating these two functions into a single one as mentioned before.